interface/patient_file/ccr_import.php

```diff
@@ -22,39 +22,34 @@
  * @author  Ajil P M <ajilpm@zhservices.com>
  * @link    http://www.open-emr.org
  */
+require_once(dirname(__FILE__) . "../../globals.php");
 
-
-
-require_once(dirname(__FILE__) . "/../globals.php");
+use OpenEMR\Core\Header;
 
 ?>
 <html>
 <head>
+<?php Header::setupHeader(['opener']); ?>
 <title><?php echo xlt('Import');?></title>
 <link rel="stylesheet" href="<?php echo $css_header; ?>" type="text/css">
```

Comment:
Remove above line since this is now being brought in via the Header::setupHeader call above